Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -1,5 +1,6 @@
 import Q = require('q');
 import tl = require('azure-pipelines-task-lib/task');
+var path = require('path');
```

Comment:
```suggestion
const path = require('path');
```
Can we replace it with 'const'?